exercises/practice/leap/leap_test.sql

```diff
@@ -0,0 +1,59 @@
+-- Create database:
+.read ./create_fixture.sql
+
+-- Read user student solution and save any output as markdown in user_output.md:
+.mode markdown
+.output user_output.md
+.read ./success.sql
```

Comment:
Is this supposed to be `leap.sql` ?

Reply:
Yes. Bad copy and paste from the test runner. Thanks for the thorough check. We need to implement ci to run tests, so we know they are fibe

Reply:
Less a thorough review, more me trying to copy/paste and reuse for raindrops :smile: